batavia/vendor/yarn.lock

```diff
@@ -0,0 +1,21 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm not 100% across what yarn brings to the table, but i'd appreciate if someone could enlighten me, or link to something that says why we should be using an extremely new, in-flux packaging system. If it gives super huge benefits over node, then fine. I just want to be informed.